conf/application.conf

```diff
@@ -185,6 +178,11 @@ microservice {
   }
 }
 
+mongodb {
+  uri = "mongodb://localhost:27017/help-to-save"
+  session.expireAfter = 1 hour
```

Comment:
is this the expiry time we currently have in prod?

Reply:
yes, they have 60 min in prod : 
https://github.com/hmrc/app-config-production/blob/master/key-store.yaml#L54